pkg/odo/util/cmdutils.go

```diff
@@ -2,12 +2,12 @@ package util
 
 import (
 	"fmt"
-	"os"
-	"strings"
-
 	"github.com/redhat-developer/odo/pkg/api"
 	"github.com/redhat-developer/odo/pkg/log"
 	"github.com/redhat-developer/odo/pkg/machineoutput"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+	"os"
```

Comment:
stdlib imports ("os" here) should appear in the first block of imports

Reply:
IDE did something. I changed some settings, and hope it's not a problem any more. At least, I couldn't find any as far as I checked. Thanks for catching. 👍🏾